install.md

```diff
@@ -118,42 +116,134 @@ yum install -y \
+
+#### ostree
+
+A copy of the development libraries for `ostree` is necessary, either in the form of the `libostree-dev` package
+from the [flatpak](https://launchpad.net/~alexlarsson/+archive/ubuntu/flatpak) PPA,
+or built [from source](https://github.com/ostreedev/ostree/blob/master/docs/contributing-tutorial.md)
+(see also [here](https://ostree.readthedocs.io/en/latest/#building)). As of Ubuntu 18.04, `libostree-dev` is available in the main repositories,
+and the PPA is no longer required.
+
+To build, use the following (running `make` can take a while):
+```bash
+git clone https://github.com/ostreedev/ostree ~/ostree
+cd ~/ostree
+git submodule update --init
+# for Fedora, CentOS, RHEL
+sudo yum install -y automake bison e2fsprogs-devel fuse-devel libtool xz-devel zlib-devel
+# for Debian, Ubuntu etc.
+sudo apt-get install -y automake bison e2fsprogs fuse liblzma-dev libtool zlib1g
+
+./autogen.sh --prefix=/usr --libdir=/usr/lib64 --sysconfdir=/etc
+# remove --nonet option due to https:/github.com/ostreedev/ostree/issues/1374
+sed -i '/.*--nonet.*/d' ./Makefile-man.am
+make
+sudo make install
+```
+
+#### golang
+
+Be careful to double-check that the version of golang is new enough (i.e. `go version`), version 1.10.x or higher is required.
+If needed, golang kits are available at https://golang.org/dl/. Alternatively, go can be built from source as follows
+(it's helpful to leave the system-go installed, to avoid having to [bootstrap go](https://golang.org/doc/install/source):
+
+```bash
+export GOPATH=~/go
+git clone https://go.googlesource.com/go $GOPATH
+cd $GOPATH
+git checkout tags/go1.10.8  # optional
+cd src
+./all.bash
+export PATH=$GOPATH/bin:$PATH
+```
+
+#### conmon
+
+The latest version of `conmon` is expected to be installed on the system. Conmon is used to monitor OCI Runtimes.
+To build from source, use the following (if not already executed above, run `export GOPATH=~/go && mkdir -p $GOPATH`):
+
+```bash
+git clone https://github.com/cri-o/cri-o $GOPATH/src/github.com/cri-o/cri-o
+cd $GOPATH/src/github.com/cri-o/cri-o
+mkdir bin
+make bin/conmon
+sudo install -D -m 755 bin/conmon /usr/libexec/podman/conmon
+```
+
+#### CNI plugins
+
+```bash
+git clone https://github.com/containernetworking/plugins.git $GOPATH/src/github.com/containernetworking/plugins
+cd $GOPATH/src/github.com/containernetworking/plugins
+./build_linux.sh
+sudo mkdir -p /usr/libexec/cni
+sudo cp bin/* /usr/libexec/cni
+```
+
+#### runc
+
+The latest version of `runc` is expected to be installed on the system. It is picked up as the default runtime by podman.
+Version 1.0.0-rc4 is the minimal requirement, which is available in Ubuntu 18.04 already.
+To double-check, `runc --version` should produce at least `spec: 1.0.1`, otherwise build your own:
+
+```bash
+git clone https://github.com/opencontainers/runc.git $GOPATH/src/github.com/opencontainers/runc
+cd $GOPATH/src/github.com/opencontainers/runc
+make BUILDTAGS="selinux seccomp"
```

Comment:
This build tag has been changed compared to the tutorial due to [this comment](https://github.com/containers/libpod/issues/3045#issuecomment-488113241).